browser/profiles/brave_profile_impl.cc

```diff
@@ -23,6 +23,9 @@ BraveProfileImpl::BraveProfileImpl(
     scoped_refptr<base::SequencedTaskRunner> io_task_runner)
     : ProfileImpl(path, delegate, create_mode, creation_time, io_task_runner),
       weak_ptr_factory_(this) {
+  if (create_mode == CREATE_MODE_SYNCHRONOUS) {
+    brave::RecordInitialP3AValues(this);
```

Comment:
can we add a comment describing why this is done here for synchronous mode, and there for for asynchronous?

Reply:
done!